### ISSUE TITLE
Reproducing examples_codes.ipynb with different libraries versions doesn't create identical spectrograms 

### ISSUE THREAD

**Issue by arachid1** [keunwoochoi/kapre#75]
Hello, i tried replicating your notebook example_codes.ipynb. 

At first, I set a kernel with the exact same versions in your notebook:

```
librosa version: 0.6.2
Keras version: 2.2.0
Keras backend: tensorflow: 1.9.0
Keras image data format: channels_last
Kapre version: 0.1.3.1
```

and I am able to reproduce your results perfectly.

However, it doesn't generate the same mel spectrogram with my version of librosa/keras/tensorflow/kapre, which are the following: 

```
2020/6/23
librosa version: 0.7.2
Keras version: 2.4.2
Tensorflow version: 2.2.0
Keras image data format: channels_last
Kapre version: 0.1.7
```

but the kapre spectrogram is just not correct. Here is the output:

![image](https://user-images.githubusercontent.com/29982236/85464434-dc8ff380-b564-11ea-9109-8a2f4a2b108b.png)

I even tried with the file you use in your notebook:

![image](https://user-images.githubusercontent.com/29982236/85464627-1eb93500-b565-11ea-84f8-186929d93249.png)

I would appreciate some help. thanks for an amazing tool!

**Comment by arachid1** [comment]
I also get the following error:

UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')

**Comment by keunwoochoi** [comment]
> following error

It's an warning msg from librosa and you can ignore it

**Comment by keunwoochoi** [comment]
The difference will be fixed in Kapre 0.3.0, which is going to be released soon.
